src/main/java/com/github/jcustenborder/kafka/connect/snmp/SnmpTrapSourceConnectorConfig.java

```diff
@@ -54,13 +58,55 @@
   static final String POLL_BACKOFF_MS_DOC = "The amount of time in ms to wait if no records are returned.";
   static final long POLL_BACKOFF_MS_DEFAULT = 250;
 
+  public static final String MPV3_ENABLED_CONF = "mpv3.enabled";
+  static final String MPV3_ENABLED_DOC = "Configuration property to enable MPv3 support";
+  static final boolean MPV3_ENABLED_DEFAULT = false;
```

Comment:
Here I'd rather default to `true`. That way the user needs to explicitly configure less secure with `false` in systems where MPV3 is not supported.